src/services/lnd/schema.ts

```diff
@@ -0,0 +1,88 @@
+import * as mongoose from "mongoose"
+
+const Schema = mongoose.Schema
+
+const hopSchema = new Schema({
+  channel: String,
+  channel_capacity: {
+    type: Number,
+    min: 0,
+  },
+  fee: {
+    type: Number,
+    default: 0,
+  },
+  fee_mtokens: String,
+  forward: {
+    type: Number,
+    min: 0,
+  },
+  forward_mtokens: String,
+  public_key: String,
+  timeout: {
+    type: Number,
+    min: 0,
+  },
+})
+
+const pathSchema = new Schema({
```

Comment:
it should work for MPP right?
There can be many paths with many hops?

Reply:
Yes exactly, we haven't been able to trigger/test mpp as yet but from the object you sent when we were testing with Loop this is exactly where all the paths end up